src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,47 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { useContext } from 'react';
+import { ErrorContext } from '../contsxts/ErrorContext';
+import { LoaderContext } from '../contsxts/LoaderContext';
+import { PeopleContext } from '../contsxts/PeopleContext';
+import { PeopleFilteredContext } from '../contsxts/PeopleFilteredContext';
```

Comment:
The import path for `PeopleFilteredContext` is incorrect. It should be '../contexts/PeopleFilteredContext' instead of '../contsxts/PeopleFilteredContext'.